src/element.js

```diff
@@ -307,6 +307,7 @@ const Element = {
       easing:
         typeof transition === 'object'
           ? 'function' in transition
+          ? console.log("function key is deprecated, please use easing key") : 'easing' in transition
```

Comment:
Let's use the built in Log component here, instead of a plain console.log. Also we can improve the debug message a bit: "The property for defining a transition's easing function has been renamed from `function` to `easing`. Please update your code."